### ISSUE TITLE
Synchronize library opening hour with website

### ISSUE THREAD

**Issue by yujinkimmn** [TUM-Dev/NavigaTUM#1087]
## Steps to Reproduce

1. Click hamburger menu icon (upper left)
2. Opening Hours category 
3. Libraries

## Problem
In some branch libraries, the opening hour in app and in website are not synchronized. 
For example, here are the differences.     

    
> Mathematics & Informatics
- In website: Mon-Fri 8-24 / Sat-Sun 10-20
- In app: Mon-Fri 9-24      

> Mechanical Engineering
- In website: Mon-Fri 8-21
- In app: Mon-Thu 8-21, Fri 8-19     

> Medicine 
- In website: Mon-Fri 8-21
- In app: Pickup of preordered books: Mon-Fri 9-20

> Chemistry 
- In website: Mon-Fri 8-21
- In app: Mon-Thu 8-21, Fri 8-19    

 <p> <image src="https://user-images.githubusercontent.com/75295578/180172259-915b3608-cdf1-49e2-b442-4c281c019c8b.png" width="300" height="650"><image src="https://user-images.githubusercontent.com/75295578/180173954-1cc8d335-8b17-4d31-b41c-40ef8c63a54a.png" width="300" height="650"/> </p> (This image is for Mathematics & Informatics example)

## Suggested Enhancement :sparkles:
I think the informations on the website are correct version(Need someone's review about this. I'm not sure.)
Through this issue, I'd like to edit and synchronize these opening hours.

**Comment by CommanderStorm** [comment]
Related https://github.com/TUM-Dev/NavigaTUM/issues/961

**Comment by CommanderStorm** [comment]
> *This was generated by AI during triage.*

Consolidated into #3050, which covers automated opening-hours ingestion across libraries, mensas, and other facilities. Your concrete examples of website-vs-app divergence (Math/Informatics, Mechanical Engineering, Medicine, Chemistry libraries) are explicitly referenced there as the canonical test cases — those should still work once the new scraper lands.

Closing in favour of the umbrella.
